include/fmt/core.h

```diff
@@ -1366,8 +1357,10 @@ inline typename std::enable_if<
   std::back_insert_iterator<Container>>::type
     format_to(std::back_insert_iterator<Container> out, const S &format_str,
               const Args &... args) {
+  typedef typename buffer_context<FMT_CHAR(S)>::type context;
+  internal::checked_args<S, Args...> ca(format_str, args...);
   return vformat_to(out, internal::to_string_view(format_str),
-                    internal::checked_args<S, Args...>(format_str, args...));
+                    basic_format_args<context>(ca));
```

Comment:
You can just do `*ca` instead of `basic_format_args<context>(ca)`.

Reply:
Cool!